src/components/table/TableCellExpander.tsx

```diff
@@ -0,0 +1,48 @@
+import { CellProps } from 'react-table';
+import { ReactComponent as IconChevronDown } from 'assets/icons/chevronDown.svg';
+import { PropsWithChildren } from 'react';
+
+interface Props {
+  isExpandedContent?: JSX.Element;
+  isCollapsedContent?: JSX.Element;
+  canNotExpandContent?: JSX.Element;
```

Comment:
I'm confused by the name `canNotExpandContent`

Reply:
done